### DISCUSSION TITLE
Using BS Swiper to Show Custom Taxonomy Archive Pages

### DISCUSSION THREAD

**Discussion by digidoda** [bootscore/bootscore#573] [Q&A]
Hi all, I'm currently using bs swiper on my site, but not sure if what I'm trying to achieve is possible so though I'd ask :)

I have a custom taxonomy setup 'caterers-range' where I have added ACF fields to allow me to add images and other info to the taxonomy.

Is it possible to output a list of all of my 'caterers-ranges' using bs swiper and display their images and link through to their archive page?

I thought I'd tried everything in the shortcode, but wondering if there was something obvious I was missing?

**Comment by crftwrk** [comment]
Hi,

you asked this already for bS Isotope. Note that bS Grid, bS Swiper and bS Isotope are plugins which are only able to show something in the loop. They have nothing to do with the archive. 

If you want to have a custom archive, it's theme area - and yes, it's possible. But I still do not understand what you exactly want to do. So, sharing a link to your project will clear this up quickly and helps to show a solution.
